mautrix_facebook/example-config.yaml

```diff
@@ -7,7 +7,6 @@ homeserver:
     # Whether or not to verify the SSL certificate of the homeserver.
     # Only applies if address starts with https://
     verify_ssl: true
-    asmux: false
```

Comment:
Due to the way config updating works, all fields that are used should be included in the base config (this field is used [here](https://github.com/tulir/mautrix-python/blob/v0.7.11/mautrix/bridge/user.py#L69))